### PR TITLE
Minor fixes and refactors to calc-cbs-params script

### PR DESCRIPTION
**Pull request by aguedes**
Hi folks,

This PR contains some minor fixes and refactors to the CBS helper scripts. No new functionality is added.

Regards,

Andre

### CODE REVIEW

File: misc/calc-cbs-params.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def calc_sr_class_params(streams):
     max_frame_size = 0
     for stream in streams:
         frame_size = calc_frame_size(stream)
-        idleslope += int(stream['rate']) * frame_size * 8 / 1000
+        idleslope += int(stream['rate']) * frame_size * 8 // 1000
```
Comment (edersondisouza) [inline misc/calc-cbs-params.py]: This is about the commit message: `stream[rate]` is a multiple of 4000 only when thinking about 802.1Q SR class A and B, but one could use this script for other profiles, where this assumption wouldn't hold. Do we care about this? Would change the commit message from "always" to "usually" be more precise?

Comment (aguedes) [reply misc/calc-cbs-params.py]: Good point! I forgot about Avnu's automotive profile which defines SR classes with 750 and 689 packet rate.

In this case, I'll simply remove this patch from the PR and fix `main()` so it rounds up `idleslope` before printing it. Currently we truncate it so we end up allocating slightly less bandwidth than what the streams need. Sounds good?

Comment (edersondisouza) [reply misc/calc-cbs-params.py]: It does!

**Comment by aguedes** [comment]
Hi @edersondisouza, just pushed a new version, please take a look. I ended up fixing `calc_sr_class_params()` instead of `main()` since we need idleslope to calculate the other CBS parameters.

**Comment by edersondisouza** [comment]
:8ball:

**Comment by aguedes** [comment]
Thanks for the review, @edersondisouza! I'm merging this now.

**Pull request merged**